pages/common/mysql.md

```diff
@@ -18,6 +18,6 @@
 
 `mysql --socket {{path/to/socket.sock}}`
 
-- Execute SQL statements in a script file (batch file):
+- Execute SQL statements in a script file (batch file), user will be prompted for a password:
 
-`mysql {{database_name}} < {{script.sql}}`
+`mysql -u {{user}} --password -e "source {{filename.sql}}" {{database_name}}`
```

Comment:
Sorry, please let's not add the user and password options back again. Those weren't there for a reason.

The tldr project is more of an explainer than a cheatsheet -- in other words, the underlying mission is not to merely provide copy-pastable snippets (like, say, commandlinefu.com) but rather to allow readers to understand the language of the commands, so that they can then derive new commands from the provided examples, or compose several simple examples into more complex ones that are useful to them.